WTWSMS/common/dynasties/01_dynasties.txt

```diff
@@ -129,17 +129,6 @@
 10424381 = {
 	name="Elemunding"
 	culture = gepid
-	coat_of_arms = {
-		template = 0
-		layer = {
-			texture = 9
-			texture_internal = 19
-			emblem = 0
-			color = 0
-			color = 0
-			color = 0
-		}
-	}
```

Comment:
👍

Reply:
The Augustulus CoA was added recently by @Abian36, but he didn't break anything here. This old reference must be wrong.
Can texture_internal = 19 from dynasties_enlil.pdn be the intended texture?

Reply:
This: 
![image](https://user-images.githubusercontent.com/29546927/52717558-a8381a80-2fa1-11e9-97d5-c148917cf257.png)